### DISCUSSION TITLE
Module positions missing

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#678] [Q&A]
Hi,
I don't know if this is due to Astroid framework updates or to Joomla update (4 -> 5), but module positions are no longer indicated in the module details.
In the list of modules their position is correctly indicated and modules are correctly positioned in the front-end, the only problem is that in the module details its position is no longer indicated and moreover no position is available in the drop-down menu.

Is anyone experiencing the same problem?

Thanks,
Lorenzo

Joomla 5.1.0
PHP 8.3.6

![j_5-module-position-1](https://github.com/templaza/astroid-framework/assets/566221/0d6d73f6-62ff-44ec-b552-cda9cca86ecf)

![j_5-module-position-2](https://github.com/templaza/astroid-framework/assets/566221/3c979b9d-5aa8-4515-ab81-765d2a04447b)

**Comment by sonvnn** [comment]
Please check your templateDetails.xml file. Make sure that position is available.

**Comment by iorbita** [comment]
... position(s) tag is available ...

![templateDetails](https://github.com/templaza/astroid-framework/assets/566221/c0f2ffb7-8924-4191-aa56-ac9f7fc6c141)

**Reply by sonvnn** [reply]
Sorry but I don't see the position named "Slide botega" in your template position. Please append it to positions fieldset.

**Comment by Chacapamac** [comment]
Weird! — Did you try to empty all caches (Astroid, Joomla, Browser)
Do you have the same problem in ALL modules?

**Comment by iorbita** [comment]
@Chacapamac yes, emptied cache for all and the problem is on all modules :(
@sonvnn to be more precise, I don't have any positions displayed, and I've created quite a few, I don't even have the ones listed in the xml file, which are the template's default positions, there are no positions in the drop-down list...

**EDIT** just for information, I've just installed an old J!4 version of the site and the new module positions I've created are visible but are not reported in templateDetails.xml file 

![j_5-module-position-3](https://github.com/templaza/astroid-framework/assets/566221/941a148d-f17e-4ff5-aef5-07e92ea4dbad)

**Reply by sonvnn** [reply]
If you would like to create new position. I recommend use child-template. It avoid to lose override code when update parent templates.

**Reply by iorbita** [reply]
thank you @sonvnn but it doesn't explain me how to recover and display my modules positions and why Astroid_template_one default positions, which are in templateDetails.xml, are also not displayed in the list of positions... finally no positions are displayed in the list, why?

**Reply by sonvnn** [reply]
Unfortulately, I don't know about this issue. I did not edit positions xml of Template_One. It's original from Joomdev. I suggest you edit layout builder and add new position in this case

**Reply by iorbita** [reply]
@sonvnn but in the layout builder positions are listed.
The problem here is that no positions are listed in the edit module, not even the default ones.
On your side, do you have the new positions listed in templateDetails.xml file?
![astroid-layout-builder](https://github.com/templaza/astroid-framework/assets/566221/3f7849f5-0479-4391-9500-a695247977a0)

**Reply by sonvnn** [reply]
Please help me try to switch to Cassiopeia and try to disable Astroid system plugin. If this issue is still happening. I think you should re-install Joomla core.

**Reply by iorbita** [reply]
I switch to Cassiopeia and disabled Astroid system plugin, but nothing change. So I re-installed Joomla core files, nothing, same problem. This problem make me crazy...

**Reply by sonvnn** [reply]
I think this issue is related to your Joomla core or 3rd party extensions. Raise this issue on Joomla forum or try disabling 3rd party extensions one by one to find out where the issue is.

**Reply by iorbita** [reply]
ok, I disabled all extensions and positions are displayed now, so it's a conflict extension, but which one? :((((

**Reply by iorbita** [reply]
At last I've activated the error report in Joomla's configuration, I'm getting this error, is it a problem with Joomla's library?

**EDIT:** ok, I found it, it's a dev version of EasyLayouts that I installed last week for a multilingual issue, thanks for your patience.


![error-php-j_5](https://github.com/templaza/astroid-framework/assets/566221/0b1b9c37-1645-497d-91dd-0647588110be)

**Reply by sonvnn** [reply]
I am glad to see you solve this issue.
